oblt-cli/cluster-create-custom/action.yml

```diff
@@ -0,0 +1,103 @@
+name: 'Oblt-cli create custom'
+description: 'Run the oblt-cli wrapper to create a custom cluster.'
+inputs:
+  template:
+    description: 'The Oblt cluster template to use'
+    required: true
+  parameters:
+    description: 'The Oblt cluster parameters to use in JSON format.'
+    required: true
+  github-token:
+    description: 'The GitHub access token.'
+    required: true
+  cluster-name-prefix:
+    description: 'Prefix to be prepended to the randomised cluster name'
+    required: false
+  cluster-name-suffix:
+    description: 'Suffix to be appended to the randomised cluster name'
+    required: false
+  slack-channel:
+    description: 'The slack channel to notify the status.'
+    default: '#observablt-bots'
+    required: false
+  username:
+    description: 'Username to show in the deployments with oblt-cli, format: [a-z0-9]'
+    default: 'apmmachine'
+    required: false
+  gitops:
+    description: 'Whether to provide the GitOps metadata to the oblt-cli'
+    default: false
+    required: false
+  dry-run:
+    description: 'Whether to dryRun'
+    default: false
+    required: false
+  skip-random-name:
+    description: 'Whether to deploy a cluster with a random name'
+    default: false
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Configure oblt-cli flags for gitops - issues
```

Comment:
Do you think we can use typescript instead?

I think https://github.com/elastic/apm-pipeline-library/blob/0e1a12b2aeca817bbe7a6fa056a4dad6b77b2dec/.github/actions/oblt-cli-create-serverless/action.yml#L40 can help with

Reply:
https://github.com/elastic/oblt-actions/pull/14/files/237d4b567cc11fb0305b32cbd5cf064aba18936e..422b5ddd8acbe64acbfea96f26e0e4ecfea4919b